src/flows/install-happ.ts

```diff
@@ -230,50 +244,64 @@ export const setupServiceLogger = async (masterClient, {hostedHappId}) => {
     agentId,
     conductorInterface: Config.ConductorInterface.Internal
   })
-  await setupHolofuelBridge(masterClient, {callerInstanceId: instanceId})
+  await setupHolofuelBridge(masterClient, {callerInstanceId: instanceId, replace: false})
 
   // TODO: make initial call to serviceLogger to set up preferences?
 }
 
-export const lookupAppEntryInHHA = async (client, {happId}: LookupHappRequest): Promise<HappEntry> => {
+export const lookupAppEntryInHHA = async (client, {happId}: LookupHappRequest): Promise<HappStoreEntry> => {
```

Comment:
This function looks like it could be more readable using standard .then expressions :)